src/main/java/hwr/oop/application/Database.java

```diff
@@ -1,34 +1,209 @@
+            String sqlFile = new String(Files.readAllBytes(Paths.get(System.getenv("SQL_FILE_URL"))));
+            preparedStatement.executeUpdate(sqlFile);
+        } catch (SQLException | IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void insertIntoProject(String title, String description, LocalDateTime dateTimeCreated, LocalDateTime dateTimeDeadline, int ownerId) {
+        Connection connectionToDatabse = connect();
+        PreparedStatement preparedStatement = null;
+        try {
+            String sql = "INSERT INTO projects (title, description, dateTimeCreated, dateTimeDeadline, ownerId) VALUES (?,?,?,?,?)";
+            preparedStatement = connectionToDatabse.prepareStatement(sql);
+
+            preparedStatement.setString(1, title);
+            preparedStatement.setString(2, description);
+            preparedStatement.setString(3, formatToString.format(dateTimeCreated));
+            preparedStatement.setString(4, formatToString.format(dateTimeDeadline));
+            preparedStatement.setInt(5, ownerId);
+            preparedStatement.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            // Close the resources
+            try {
+                if (preparedStatement != null) {
+                    preparedStatement.close();
+                }
+                if (connectionToDatabse != null) {
+                    connectionToDatabse.close();
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public void insertIntoTasks(String description, int status, int priority, LocalDateTime dateTimeCreated, LocalDateTime dateTimeDone, LocalDateTime dateTimePlannedStart, LocalDateTime dateTimePlannedEnd, LocalDateTime dateTimeDeadline, int projectId) {
+        Connection connectionToDatabase = connect();
+        PreparedStatement preparedStatement = null;
+        try {
+            String sql = "INSERT INTO tasks (description, status, priority, dateTimeCreated, dateTimeDone, dateTimePlannedStart, dateTimePlannedEnd, dateTimeDeadline, projectId) VALUES (?,?,?,?,?,?,?,?,?)";
+            preparedStatement = connectionToDatabase.prepareStatement(sql);
+
+            preparedStatement.setString(1, description);
+            preparedStatement.setInt(2, status);
+            preparedStatement.setInt(3, priority);
+            preparedStatement.setString(4, formatToString.format(dateTimeCreated));
+            preparedStatement.setString(5, formatToString.format(dateTimeDone));
+            preparedStatement.setString(6, formatToString.format(dateTimePlannedStart));
+            preparedStatement.setString(7, formatToString.format(dateTimePlannedEnd));
+            preparedStatement.setString(8, formatToString.format(dateTimeDeadline));
+            preparedStatement.setInt(9, projectId);
```

Comment:
```suggestion
            preparedStatement.setInt(9, task.getProject().getID());
```
Methode für getID muss noch implementiert werden.